README.md

```diff
@@ -1,5 +1,30 @@
 > [!NOTE]
-> This is a fork of [oxc-resolver] and [rspack-resolver](https://github.com/web-infra-dev/rspack-resolver), cause 100% compatible with [enhanced-resolve] is the non-goal of [oxc-resolver] itself, we may add [enhanced-resolve] specific features like [`pnp support`](https://github.com/web-infra-dev/rspack/issues/2236) and [`alternative support`](https://github.com/web-infra-dev/rspack/issues/5052) in the future.
+>
+> This is a fork of [oxc-resolver] and [rspack-resolver], and will be used in [eslint-plugin-import-x] and [eslint-import-resolver-typescript] cause 100% compatible with [enhanced-resolve] is the non-goal of [oxc-resolver] itself, we add [enhanced-resolve] specific features like [`pnp support`](https://github.com/web-infra-dev/rspack/issues/2236).
```

Comment:
Suggestion: Split into two sentences. One concerned with forking from the projects and another for compatibility with `enhanced-resolve`, the latter mentioning that `rspack-resolver` has that same goal.

Maybe something like:

```md
We align with [rspack-resolver] on being compatible with [enhanced-resolve] – a non-goal for [oxc-resolver].
```

Reply:
Would you like to raise a PR instead? Actually I'm not a native English speaker, so there must be some words/sentences could be improved.